pyvrp/cpp/search/Route.h

```diff
@@ -448,18 +457,35 @@ class Route
      */
     [[nodiscard]] inline Cost unitDurationCost() const;
 
+    /**
+     * @return Cost per unit of overtime on this route.
+     */
+    [[nodiscard]] inline Cost unitOvertimeCost() const;
+
     /**
      * Returns true if this route has duration-related cost components, either
      * via the objective or via penalised constraints. False otherwise.
      */
     [[nodiscard]] inline bool hasDurationCost() const;
 
     /**
-     * @return The maximum route duration that the vehicle servicing this route
-     *         supports.
+     * @return The (soft) maximum shit duration that the vehicle servicing this
+     *         route supports. This may optionally be extended with overtime.
```

Comment:
```suggestion
     * @return The (soft) maximum shift duration that the vehicle servicing this
     *         route supports. This may optionally be extended with overtime.
```
💩

Reply:
I see no problem here 😆